### ISSUE TITLE
Readme Error: No sub, no way to Unsub()

### ISSUE THREAD

**Issue by jraedisch** [nbd-wtf/go-nostr#17]
The README.md example seems broken. There is no sub, just subId. Would submit PR but did not find a good way to unsub.

Referenced part of the README.md:

```
subId, events := pool.Sub(nostr.Filters{
	{
		Authors: []string{"0ded86bf80c76847320b16f22b7451c08169434837a51ad5fe3b178af6c35f5d"},
		Kinds:   []int{nostr.KindTextNote}, // or {1}
	},
})

go func() {
	for event := range nostr.Unique(events) {
		log.Print(event)
	}
}()

time.Sleep(5 * time.Second)
sub.Unsub()
```

**Comment by stanstacks** [comment]
You haven't figured it out yet have you? I just updated from 0.7.4 => 0.9.0 and trying to figure this out too. Subscriptions seem to be relay based rather than pool based now:

https://github.com/nbd-wtf/go-nostr/blob/2d01aa863053085cf0097a63a1cacf2a7951491b/relay.go#L214

@fiatjaf first branle, and now this?

**Comment by fiatjaf** [comment]
The bug is acknowledged and will be fixed.

I don't understand what you're tagging me and talking about branle. If you are unhappy with the code please send a pull request fixing this or wait until I get to it.

**Comment by stanstacks** [comment]
Sometimes I just like to tease you.

Fixed in #20 I hope

**Comment by stanstacks** [comment]
@jraedisch this is solved, can you confirm and close?
